### PR TITLE
feat: improve the vue textbox component

### PR DESCRIPTION
**Pull request by AliKdhim87**

### CODE REVIEW

File: packages/storybook-vue/src/stories/Textbox/TextboxWithFormBinding.stories.mdx [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  setup() {
+    return { args };
+  },
+  template: `
+  <FormField> 
+    <Textbox v-bind="args" @update:modelValue="action" :value="args.modelValue" />
+  <br/>
+  <br/>
+  <Paragraph v-if="args.modelValue"><strong> Textbox Binding:</strong> {{args.modelValue}}</Paragraph>
+  </FormField>
```
Comment (Robbert) [inline packages/storybook-vue/src/stories/Textbox/TextboxWithFormBinding.stories.mdx:22]: These comments from the textarea PR also apply here:

- https://github.com/nl-design-system/utrecht/pull/1107/files#r935427944
- https://github.com/nl-design-system/utrecht/pull/1107/files#r935425726

File: packages/storybook-vue/src/stories/Textbox/TextboxTypes.stories.mdx [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+          code: `<Textbox placeholder="Textbox Number" name="textbox-number" id="textbox-number" type="number"/>`,
+        },
+      },
+    }}
+  >
+    {Template.bind({})}
+  </Story>
+</Canvas>
+
+<ArgsTable of={Textbox} />
```
Comment (Robbert) [inline packages/storybook-vue/src/stories/Textbox/TextboxTypes.stories.mdx:111]: I agree it is nice and useful to display the `type` variants here, but for completeness sake I think we should use this PR to update the textbox implementation to render all the relevant class names. You can use the React component as reference:

https://github.com/nl-design-system/utrecht/blob/main/packages/component-library-react/src/Textbox.tsx#L33-L38

File: packages/storybook-vue/src/stories/Textbox/TextboxWithFormBinding.stories.mdx (hunk 1 above)
Comment (AliKdhim87) [reply packages/storybook-vue/src/stories/Textbox/TextboxWithFormBinding.stories.mdx:22]: Regarding the second comment, I already [answered](https://github.com/nl-design-system/utrecht/pull/1107/files#r935510274), I need time to find a solution or maybe Storybook will release a new version that will solve this issue

File: packages/storybook-vue/src/stories/Textbox/TextboxTypes.stories.mdx (hunk 2 above)
Comment (AliKdhim87) [reply packages/storybook-vue/src/stories/Textbox/TextboxTypes.stories.mdx:159]: I created a [PR](https://github.com/nl-design-system/utrecht/pull/1119) to enable the related style from the css

**Review by Robbert — APPROVED** [review]

**Pull request merged**